app/api/api_v1/routers/search.py

```diff
@@ -140,19 +140,23 @@ def search_documents(
     the search database. The continuation token can be used to get the next set of
     results from the search database. See the request schema for more details.
     """
+    origin = request.headers.get("origin")
+    if origin is not None:
```

Comment:
Assume if origin is None then we'll throw on 159?

Reply:
We should perhaps test this as part of `test_decoding_token_with_invalid_aud_raises_expired_signature_token_error` ?

Reply:
If the audience is None here, decode_config_token will throw an `InvalidAudienceError` which is captured under `test_search_decoding_token_raises_PyJWTError`

Reply:
That test confirms the http response given a thrown error. Not ... if the origin is none the error is thrown

Or am I missing something?

Reply:
That's fair, I'll patch that in a new test :)